python/kserve/kserve/storage.py

```diff
@@ -221,7 +221,7 @@ def _download_blob(uri, out_dir: str):  # pylint: disable=too-many-locals
                 else:
                     blobs += container_client.list_blobs(name_starts_with=item.name,
                                                          include=['snapshots'])
-        for blob in set(blobs):
+        for blob in blobs:
```

Comment:
@maganaluis Can you help review here? not sure if we had `set` there for any reason.

Reply:
@yuzisun It should be okay to remove it. Looking back to the upgrade to 12.X.X there were a lot of changes to the mock unit-tests, I don't think there should be duplicates keys coming from the client. So this must have been to handle those unit-tests.